src/app/containers/ArticleTimestamp/helpers.test.js

```diff
@@ -117,8 +125,8 @@ describe('ArticleTimestamp helper functions', () => {
     });
 
     it(`should return date format when lastPublished and firstPublished are not on the same day`, () => {
-      const firstPublished = timestampGenerator({ hours: 50 });
-      const lastPublished = timestampGenerator({ hours: 20 });
+      const firstPublished = timestampGenerator({ days: 4 });
+      const lastPublished = timestampGenerator({ days: 2 });
```

Comment:
@jamesdonoh, since this one read `should return date format when lastPublished and firstPublished are not on the same day` I thought it'd be much clearer to distinguish by days not hours

Reply:
Agreed, thanks for explanation.